src/main/java/edu/harvard/iq/dataverse/authorization/DataverseRole.java

```diff
@@ -117,15 +119,29 @@ public void setId(Long id) {
 	}
 
 	public String getName() {
-		return name;
+		if (alias != null )
+		{
+			String key = "role." + alias.toLowerCase() +".name";
+			return BundleUtil.getStringFromPropertyFile(key,"BuiltInRoles" );
```

Comment:
How will this work with Custom roles?

Reply:
@scolapasta:  I didn't handle custom role in this pull request.  Is custom role also comes from JSON files?  Haven't investigated on the logic behind custom role.  Please add some information related to custom roles,  can we handle it in separate issue?

Reply:
After testing the fix @JayanthyChengan made today, custom roles in our local dvdev is now working as expected, can create/edit/apply custom roles. 
![customroles](https://user-images.githubusercontent.com/17624834/47119209-09402380-d238-11e8-9f0b-9b0ca6283ec7.png)
![applyingmytestrole](https://user-images.githubusercontent.com/17624834/47119212-0ba27d80-d238-11e8-85bb-e4376060a0f5.png)

Reply:
@scolapasta: the code got outdated after I committed new code( which will handle custom roles).  Hope I understood correctly and updated the code accordingly.  Please review